module/platform/Webots/src/Webots.cpp

```diff
@@ -256,8 +256,10 @@ namespace module::platform {
                 // Because of this, we move the servo as fast as we can to reach the position.
                 // The fastest speed is determined by the config, which comes from the max servo velocity from
                 // NUgus.proto in Webots
-                double speed =
-                    duration.count() > 0 ? diff / std::chrono::duration<double>(duration).count() : max_velocity;
+                double speed = duration.count() > 0
```

Comment:
This shouldn't have changed so maybe keep the old one?

Reply:
@TrentHouliston which cast do you like better? Check previous comment